src/clr-angular/timeline/_timeline.clarity.scss

```diff
@@ -22,24 +22,24 @@
       min-height: $clr-timeline-icon-size;
     }
 
-    clr-icon[shape='spinner'] {
-      animation: spin 1s linear infinite;
-    }
+    //clr-icon[shape='spinner'] {
```

Comment:
why are you commenting this out?

Reply:
The example provided uses a `spinner` icon that doesn't exist in Clarity icons (it was added via `ClarityIcons.add()` api. 

I had to replace it with the native clr-spinner in clarity which already has the appropriate styles. I did forget to remove the unused code so will push a change for that.